ann_benchmarks/algorithms/hdidx_.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
```

Comment:
Why the trailing underscore?

Reply:
it was to avoid import naming collisions, but I see now that `absolute_import` gets around the issue.